src/test/app/Discrepancy_test.cpp

```diff
@@ -0,0 +1,4089 @@
+        // create a temporary path to write DB files to and
+        // our starting ledger data
+        auto path  = boost::filesystem::temp_directory_path(ec);
+        BEAST_EXPECTS(!ec, ec.message());
+        path /= boost::filesystem::unique_path("%%%%-%%%%-%%%%-%%%%", ec);
+        BEAST_EXPECTS(!ec, ec.message());
+        boost::filesystem::create_directories(path, ec);
+        BEAST_EXPECTS(!ec, ec.message());
+
+        auto ledgerFile = path / "ledgerdata.json";
+        std::ofstream o (ledgerFile.string(), std::ios::out | std::ios::trunc);
+        for(auto p : ledgerXRPDiscrepancyData)
+            o << p;
+        o.close();
+
+        {
+            Env env(*this, [&]()
+                {
+                    auto p = std::make_unique<Config>();
+                    test::setupConfigForUnitTests(*p);
+                    p->START_LEDGER = ledgerFile.string();
+                    p->START_UP = Config::LOAD_FILE;
+                    p->legacy("database_path", path.string());
+                    return p;
+                }());
+
+
+            Json::Value jrq;
+            jrq[jss::id] = 1;
+            jrq[jss::tx_blob] =
+                "1200002200020000240000124E61D5438D7EA4C680000000000000000000000"
+                "000004A50590000000000E5C92828261DBAAC933B6309C6F5C72AF020AFD468"
+                "400000000000000A69D4D3E7809B4814C8000000000000000000000000434E5"
+                "9000000000041C8BE2C0A6AA17471B9F6D0AF92AAB1C94D5A25732103FC5F96E"
+                "A61889691EC7A56FB2B859B600DE68C0255BF580D5C22D02EB97AFCE47447304"
+                "5022100D14B60BC6E01E5C19471F87EB00A4BFCA16D039BB91AEE12DA1142E8C"
+                "4CAE7C2022020E2809CF24DE2BC0C3DCF1A07C469DB415F880485B2B323E5B5A"
+                "A1D9F6F22D48114AFD96601692A6C6416DBA294F0DA684675A824B28314AFD96"
+                "601692A6C6416DBA294F0DA684675A824B201123000000000000000000000000"
+                "04A50590000000000E5C92828261DBAAC933B6309C6F5C72AF020AFD401E5C92"
+                "828261DBAAC933B6309C6F5C72AF020AFD4FF100000000000000000000000000"
+                "000000000000000300000000000000000000000004A50590000000000E5C9282"
+                "8261DBAAC933B6309C6F5C72AF020AFD401E5C92828261DBAAC933B6309C6F5C"
+                "72AF020AFD4FF01A034782E2DBAC4FB82B601CD50421E8EF24F3A00100000000"
+                "000000000000000000000000000000000300000000000000000000000004A505"
+                "90000000000E5C92828261DBAAC933B6309C6F5C72AF020AFD401E5C92828261"
+                "DBAAC933B6309C6F5C72AF020AFD400";
+            auto jrr = env.rpc ("json", "submit", to_string(jrq))[jss::result];
+            auto hash = jrr[jss::tx_json][jss::hash];
+            env.close();
+            Json::Value jrq2;
+            jrq2[jss::binary] = false;
+            jrq2[jss::transaction] = hash;
+            jrq2[jss::id] = 3;
+            jrr = env.rpc ("json", "tx", to_string(jrq2))[jss::result];
+            uint64_t fee = jrr[jss::Fee].asUInt();
```

Comment:
I'd hope that this line wouldn't compile, but it does.  Since I don't see a `using namespace std`, I'd hope this type would need to be spelled `std::uint64_t`.   But it compiles without the namespace qualifier.  Any ideas why?  I generated a preprocessed file and did a quick search for unscoped `using namespace std;`.  I didn't spot any, so I'm puzzled what's up.  I suspect the problem, whatever it is, is outside your file.  So I'm not asking you to fix it.  I'm just puzzled.